internal/kibana/synthetics/update.go

```diff
@@ -33,7 +33,12 @@ func (r *Resource) Update(ctx context.Context, request resource.UpdateRequest, r
 		return
 	}
 
-	namespace := plan.SpaceID.ValueString()
+	// Use namespace if explicitly set, otherwise fall back to space_id
+	namespace := plan.Namespace.ValueString()
+	if namespace == "" || plan.Namespace.IsNull() || plan.Namespace.IsUnknown() {
+		namespace = plan.SpaceID.ValueString()
+	}
+
```

Comment:
```suggestion
```

The monitor should only ever exist in the space defined by `space_id`, can you revert this change.